fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/ContainmentIndexImpl.java

```diff
@@ -393,50 +406,56 @@ private NamedParameterJdbcTemplate getNamedParameterJdbcTemplate() {
         return new NamedParameterJdbcTemplate(getDataSource());
     }
 
+    public void setContainsLimit(final int limit) {
```

Comment:
Is this just for testing purposes? If so, it could probably be package-private and optionally annotated with `@VisibleForTesting`